boot.php

```diff
@@ -11,12 +11,15 @@
 
 namespace FriendsOfRedaxo\QuickNavigation;
 
-use FriendsOfRedaxo\QuickNavigation\Buttons\ArticleHistory;
-use FriendsOfRedaxo\QuickNavigation\Buttons\ArticleNavButton;
-use FriendsOfRedaxo\QuickNavigation\Buttons\CatsButton;
-use FriendsOfRedaxo\QuickNavigation\Buttons\FavsButton;
+use FriendsOfRedaxo\QuickNavigation\ApiFunction\MenuRender;
+use FriendsOfRedaxo\QuickNavigation\Buttons\ArticleHistoryButton;
```

Comment:
Vorgeschlagen hatte ich Einzahl und keine Mehrzahl zu verwenden. Wenn es für dich dennoch passt, lass es gern.

```suggestion
use FriendsOfRedaxo\QuickNavigation\Button\ArticleHistoryButton;
```

Betrifft auch andere Stellen.

Reply:
Dann passe oben bitte deinen Kommentar an. Der Vorschlag ist ja so nicht von mir ;)

> Ich finde es verständlicher wenn klar ist, dass es mehrere Buttons gibt.

falls du irgendwann eine zweite API Klasse hast, wirst du dann den Namespace auch auf Mehrzahl anpassen?

Reply:
Es wird Button